src/journal/journald-server.c

```diff
@@ -614,19 +606,10 @@ static int vacuum_offline_user_journals(Server *s) {
                                 NULL,
                                 &f);
                 if (r < 0) {
-                        log_warning_errno(r, "Failed to read journal file %s for rotation, trying to move it out of the way: %m", full);
-
-                        r = journal_file_dispose(dirfd(d), de->d_name);
-                        if (r < 0)
-                                log_warning_errno(r, "Failed to move %s out of the way, ignoring: %m", full);
-                        else
-                                log_debug("Successfully moved %s out of the way.", full);
-
+                        log_warning_errno(r, "Failed to read journal file %s for rotation, ignoring: %m", full);
```

Comment:
Preexisting condition, but since you're touching the line: "failed to read" sounds strange. Shouldnt' this just be "failed to open" ?